docs/logs/field-types.md

```diff
@@ -0,0 +1,106 @@
+You can see field types that you can chose in Field Editor when adding or modifying a field.
+
+## Float/Double
+
+Use this type for all fields that contain floating point numerical values. For
+example, a field with request latency values should use Float type.
+Logsene detects automatically all floating point numbers as double precision
+numbers.
+
+
+**Please note:**
+
+
+Sometimes fields that carry measurements also include units or other
+**non-numerical** data. Imagine an example of a latency field. It may
+contain values like 200ms and a size field may contain values like 5KB.
+Do not use Float type for such fields (unless you don't extract a numeric value
+inside log shipper). You can use Analyzed String or Non-Analyzed String types,
+but note that this will prevent you from applying functions like min, max, avg, etc.
+on such field.
+A better option is to configure your log shipper to strip non-numeric portion and
+ship only numerical values. This will allow you to use this field for aggregations and reports.
+
+
+Example of a document containing a floating point number:
+
+    curl -XPOST https://logsene-receiver.sematext.com/<token>/example/ -d '{
```

Comment:
Things like this we put in ``` blocks.

Reply:
But this is code block and is interpreted as such (see the indent)

Reply:
Oh, right, indenting does that, too, I forgot.  👍